core/src/components/side-menu/side-menu-overlay/side-menu-overlay.scss

```diff
@@ -1,8 +1,15 @@
 :host {
-  position: absolute;
+  position: fixed;
   width: 100%;
   height: 100%;
   background-color: black;
   transition: opacity 0.4s linear;
   opacity: 0;
 }
+
+div {
+  position: fixed;
+  width: 100%;
+  height: 100%;
+  pointer-events: none;
```

Comment:
Did you test this with adding a element behind and try pressing it? And also with the combination of closing the side-menu but not triggering a click on the element behind?

Reply:
I made small change in this commit: https://github.com/scania-digital-design-system/tegel/pull/243/commits/8582d6e11ac836f394c11dc0bda3e156f5875455 
in order to test it, it works as intended. However, a proper test would be nice to conduct by packing and testing in react demo or angular one.

Reply:
Post-discussion comment: Functionality work as intended. Will be tested with npm pack in tegel-react-demo page.